src/open_company/app.clj

```diff
@@ -32,19 +41,22 @@
    c/hot-reload      wrap-reload
    c/dsn             (sentry-mw/wrap-sentry c/dsn)))
 
+(declare handler)
+(when c/prod?
+  (timbre/set-config! c/log-config)
+  (timbre/info "Starting production system without HTTP server")
+  (def handler
+    (-> (components/oc-system {:handler-fn app})
+        (dissoc :server)
+        component/start
+        (get-in [:handler :handler])))
+  (timbre/info "Started"))
```

Comment:
Because nginx-clojure requires the handler to be bound to a var we need to globally define it. This is somewhat anti-component but it seems fine in this case. We use a configuration parameter to trigger this definition so it's not done unless explicitly requested